frontend/lib/requests/lesson.ts

```
@@ -37,3 +45,273 @@ export async function getLessonBySlug<T extends Partial<Lesson> = Lesson>(
+      error: "Database Error: Failed to update droplet.",
+      data: null,
+    };
+  }
+}
+
+export async function revalidateLesson() {
+  revalidateTag("lesson");
+  revalidatePath("(editing)/draft/d/[slug]/[lessonSlug]", "page");
+}
+
+const CreateLessonSchema = LessonSchema.pick({
```

ESLint

[error] 'CreateLessonSchema' is assigned a value but only used as a type.

(@typescript-eslint/no-unused-vars)